### ISSUE TITLE
Disallow starting new replace/connect/disconnect/ extension or recovery phrase if any of such transactions in progress

### ISSUE THREAD

**Issue by DmitryBespalov** [gnosis/safe-ios#600]
**Is your feature request related to a problem? Please describe.**
When a "replace extension" transaction is pending, the app still allows to start yet another replace extension transaction from the menu. This leads to the incorrect wallet state, since the new transaction is based on the old wallet state (because the pending transaction did not take effect).

**Describe the solution you'd like**
Disable menu items whenever a transaction with any type of (replace/connect/disconnect extension, or replace recovery phrase) is in progress. Enable the relevant menu items when transaction is processed.

**Describe alternatives you've considered**

- Keep the items enabled, but disable "Start" button and show error message on the start screen.

  **Advantage**: allows interaction. 

  **Disadvantage**: in some cases, for example, after "disconnect extension" transaction is submitted, the user would be able to select "replace extension" in the menu and open the start screen. When the "disconnect" transaction finishes, the "replace" flow should not be accessible anymore.

- Hide the items from the menu while the transaction is pending. 

  **Advantage**: don't know. 

  **Disadvantage**: this "removes" functionality from the menu, I think it is better to show user what's possible, just disable things that are not relevant. (Similar case: on desktop, some menu items are disabled if they are not applicable).

**Comment by posthnikova** [comment]
I like the version where we keep menu items enabled but show an error on start screen and disable Start button.

![01_connect_browser_extension copy](https://user-images.githubusercontent.com/45787536/53632561-9dbe8780-3c15-11e9-8ba3-2534d45f5337.png)

**Comment by DmitryBespalov** [comment]
This variation of the error display is time-consuming and not consistent with the ones we have already - is it fine if the error displayed below balance? We do this already for generic errors, like "couldn't load the data"

**Comment by posthnikova** [comment]
Since this requires additional development, would it be easier to display an error alert when the user taps on menu item or on Start button?

The error below balances is related to balances and not the whole screen.

**Comment by DmitryBespalov** [comment]
yes, it would be easier, but it is too late for the 1.1.0 version
